lib/mpl_toolkits/tests/test_axisartist_angle_helper.py

```diff
@@ -35,6 +35,31 @@
     ''',
     re.VERBOSE
 )
+HMS_RE = re.compile(
```

Comment:
Since I've found `re.escape`, I'm going to remove this second copy the next time I update this PR.